### DISCUSSION TITLE
FingerPaint v1.2.3

### DISCUSSION THREAD

**Discussion by Wazzaps** [Wazzaps/fingerpaint#13] [Announcements]
Added hotfix for Arch/Manjaro based distros + Verify pillow version

Arch/Manjaro problem was that it doesn't set `/org/gnome/desktop/peripherals/touchpad/send-events` to any value by default. I assume it's "enabled" if it's empty now

<hr /><em>This discussion was created from the release <a href='https://github.com/Wazzaps/fingerpaint/releases/tag/v1.2.3'>FingerPaint v1.2.3</a>.</em>

**Comment by Wazzaps** [comment]
@jhossbach Could you bump the AUR version?

**Reply by jhossbach** [reply]
Will do.
